ceph_deploy/hosts/suse/install.py

```diff
@@ -106,6 +115,13 @@ def mirror_install(distro, repo_url, gpg_url, adjust_repos):
         distro.conn.remote_module.write_file(
             '/etc/zypp/repos.d/ceph.repo',
             ceph_repo_content)
+        remoto.process.run(
+            distro.conn,
+            [
+                'zypper',
+                'ref'
```

Comment:
hate to nitpick, but can this one be like the other call, using "--non-interactive" and spelling "refresh" out in full?